qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestBase.java

```diff
@@ -34,35 +30,23 @@
 import org.junit.rules.TestRule;
 import org.junit.rules.TestWatcher;
 import org.junit.runner.Description;
-import org.openmrs.contrib.qaframework.helper.TestData.EncounterInfo;
-import org.openmrs.contrib.qaframework.helper.TestData.PatientInfo;
-import org.openmrs.contrib.qaframework.helper.TestData.RoleInfo;
 import org.openmrs.contrib.qaframework.helper.TestData.TestPatient;
-import org.openmrs.contrib.qaframework.helper.TestData.UserInfo;
-import org.openqa.selenium.By;
-import org.openqa.selenium.NotFoundException;
-import org.openqa.selenium.OutputType;
-import org.openqa.selenium.TakesScreenshot;
-import org.openqa.selenium.UnexpectedAlertBehaviour;
-import org.openqa.selenium.WebDriver;
+import org.openmrs.contrib.qaframework.helper.TestData.*;
+import org.openqa.selenium.*;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeDriverService;
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.firefox.FirefoxOptions;
-import org.openqa.selenium.remote.CapabilityType;
-import org.openqa.selenium.remote.DesiredCapabilities;
-import org.openqa.selenium.remote.RemoteWebDriver;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.saucelabs.common.SauceOnDemandAuthentication;
-import com.saucelabs.common.SauceOnDemandSessionIdProvider;
-import com.saucelabs.junit.SauceOnDemandTestWatcher;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
 
-import jakarta.ws.rs.ProcessingException;
-import jakarta.ws.rs.ServerErrorException;
-import jakarta.ws.rs.client.ClientBuilder;
-import jakarta.ws.rs.core.Response;
+import static org.junit.Assert.assertTrue;
```

Comment:
are all these changes part of the ticket requirements?

Reply:
sorry updated as required